### PR TITLE
docs: Set the CA certificate for etcd in mtls.md

### PR DESCRIPTION
**Pull request by tzssangglass**
### What this PR does / why we need it:

Add about setting CA certificate for etcd in `mtls.md`.

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/en/latest/mtls.md [hunk 1]
```diff
@@ -75,6 +75,14 @@ etcd:
     key: /data/certs/etcd_client.key        # path of key used by the etcd client
 ```
 
+Also set the CA certificate.
+
+```yaml
+apisix:
```
Comment (tokers) [inline docs/en/latest/mtls.md:81]: What about just coalesce this code block and the last one (`etcd.tls`).

File: docs/en/latest/mtls.md [hunk 2]
```diff
@@ -75,6 +75,14 @@ etcd:
...
+  ssl:
+    ssl_trusted_certificate: /path/to/certs/ca-certificates.crt       # path of CA certificate used by the etcd client
```
Comment (spacewander) [inline docs/en/latest/mtls.md:83]: this is the CA used by the etcd server

Comment (tzssangglass) [reply docs/en/latest/mtls.md]: update

File: docs/en/latest/mtls.md [hunk 3]
```diff
@@ -75,6 +75,14 @@ etcd:
     key: /data/certs/etcd_client.key        # path of key used by the etcd client
 ```
 
+Also set the CA certificate.
```
[the remark this answers is not in the document]
Comment (tzssangglass) [reply docs/en/latest/mtls.md]: update expressions

File: docs/en/latest/mtls.md (hunk 1 above)
Comment (tzssangglass) [reply docs/en/latest/mtls.md:81]: put them together may be unclear

**Review by spacewander — APPROVED** [review]

File: docs/en/latest/mtls.md [hunk 4]
```diff
@@ -66,7 +66,7 @@ curl --cacert /data/certs/mtls_ca.crt --key /data/certs/mtls_client.key --cert /
 
 ### How to configure
 
-You need to [build APISIX-Openresty](./how-to-build.md#6-build-openresty-for-apisix) and configure `etcd.tls` section if you want APISIX to work on an etcd cluster with mTLS enabled.
+You need to [build APISIX-Openresty](./how-to-build.md#step-6-build-openresty-for-apache-apisix) and configure `etcd.tls` section if you want APISIX to work on an etcd cluster with mTLS enabled,
```
Comment (yzeng25) [inline docs/en/latest/mtls.md:69]: "enabled," --> "enabled." or "enabled:"

Comment (tzssangglass) [reply docs/en/latest/mtls.md]: update

**Review by tokers — APPROVED** [review]

**Pull request merged**